pkg/crc/preflight/preflight_linux.go

```diff
@@ -129,7 +129,8 @@ func checkVsock() error {
 	if err != nil {
 		return err
 	}
-	if !strings.Contains(getcap, "cap_net_bind_service+eip") {
+	if !strings.Contains(getcap, "cap_net_bind_service+eip") ||
+		!strings.Contains(getcap, "cap_net_bind_service=eip") {
```

Comment:
Shouldn't it be `&&` rather than `||`? If the string is not `cap_net_bind_service+eip` and the string is not ` cap_net_bind_service=eip` then return an error.

Reply:
@cfergeau Yes, you are right. it should `&&` operator.